packages/hub-ui/src/features/learning-forms/FormItem/FormItem.tsx

```diff
@@ -176,9 +176,18 @@ const CheckboxQuestion = (props: FormItemProps) => {
   const responses = props.responses || [];
   const values: { [key: string]: boolean } = {};
   const ref = React.useRef<HTMLInputElement>(null);

```

Comment:
It looks like we're no longer using the ref we may be able to remove the constant altogether